clcache.py

```diff
@@ -56,6 +56,7 @@
 #   value: key in the cache, under which the object file is stored
 Manifest = namedtuple('Manifest', ['includeFiles', 'includesContentToObjectMap'])
 
+CompilerArtifacts = namedtuple('CompilerArtifacts', ['objectFilePath', 'compilerStdout', 'compilerStderr'])
```

Comment:
Sure - I was under the (apparently misguided) assumption that `stdout` and `stderr` are reserved identifiers or so, but apparently not so!

That reminds me though that using `dir` is a builtin function (i.e. it's imported by default since `'dir` in dir(__builtins__)` is true), maybe not a good choice for the `Cache.dir` field.